packages/NodeTypeResolver/ValueObject/OldToNewType.php

```diff
@@ -16,11 +16,11 @@ public function __construct(
 
     public function getOldType(): Type
     {
-        return $this->oldType;
+        return $this->type;
     }
 
     public function getNewType(): Type
     {
-        return $this->newType;
+        return $this->type;
     }
 }
```

Comment:
This is a bug

Reply:
yes, need to be checked one of these example:

```diff
rules/CodingStyle/ValueObject/NameAndParent.php
    ---------- begin diff ----------
--- Original
+++ New
@@ @@
      * @param Name|Identifier $nameNode
      */
     public function __construct(
-        private Node $nameNode,
-        private Node $parentNode
+        private Node $node,
+        private Node $node
     ) {
     }

-    /**
-     * @return Name|Identifier
-     */
-    public function getNameNode(): Node
+    public function getNameNode(): Name|Identifier
     {
-        return $this->nameNode;
+        return $this->node;
     }

     public function getParentNode(): Node
     {
-        return $this->parentNode;
+        return $this->node;
     }
 }
    ----------- end diff -----------

Applied rules:
 * ArgumentAdderRector
 * UnionTypesRector
 * ReturnTypeDeclarationRector (https://wiki.php.net/rfc/scalar_type_hints_v5)
 ```

Reply:
It seems on `RenamePropertyToMatchTypeRector`